Makefile

```diff
@@ -5,7 +5,7 @@ SOURCE_GIT_IMAGE := packit
 PACKIT_TESTS_IMAGE := packit-tests
 
 build: recipe.yaml
-	ansible-bender build --build-volumes $(CURDIR):/src:Z -- ./recipe.yaml $(BASE_IMAGE) $(SOURCE_GIT_IMAGE)
```

Comment:
You can remove the `BASE_IMAGE := registry.fedoraproject.org/fedora:29` (first line) then.

Reply:
k